### PR TITLE
Update sending email settings to include MS Office 365

### PR DESCRIPTION
**Pull request by johannahemminger**
https://mendix.atlassian.net/browse/TW-1413

### CODE REVIEW

**Comment by johannahemminger** [comment]
@MarkvanMents this is the task that was on your backlog, look OK with just this small change?

File: content/en/docs/developerportal/deploy/mendix-cloud-deploy/sending-email.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ aliases:
 ---
 
 {{% alert color="warning" %}}
-Our Mendix Cloud V3 is deprecated, currently in a grace period, and will be retired at the beginning of Q3 2021. To continue running your licensed Mendix application on the Mendix Cloud, you need to migrate your app to Mendix Cloud V4. To learn more about Mendix Cloud V4 and how to migrate from Mendix Cloud V3, please visit the following page: [Migrate to Mendix Cloud V4](/developerportal/deploy/migrating-to-v4/). 
+Our Mendix Cloud V3 is deprecated. To continue running your licensed Mendix application on the Mendix Cloud, you need to migrate your app to Mendix Cloud V4. To learn more about Mendix Cloud V4 and how to migrate from Mendix Cloud V3, please visit [Migrate to Mendix Cloud V4](/developerportal/deploy/migrating-to-v4/).
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/mendix-cloud-deploy/sending-email.md:15]: Thanks for updating this 😀

**Review by MarkvanMents — APPROVED** [review]
Looks good @johannahemminger 

When I see how simple the change is I'm embarrassed that it sat on my backlog for so long!
Can you close the Jira issue (TW-1413) once this is merged.

Thanks.

**Comment by johannahemminger** [comment]
No worries @MarkvanMents !

**Pull request merged**